content/en/continuous_integration/guides/flaky_test_management.md

```diff
@@ -26,6 +26,8 @@ The app helps you prioritize flaky tests by providing the following information
 
 Once you identify a flaky test you want to fix, click on the test to see links to view the most recent failed test run or the first flaky test run.
 
+<div class="alert alert-info"><strong>Note</strong>: The table is limited to the 1000 flaky tests that exhibited more flakiness in the selected time window.</div>
```

Comment:
unsure what "exhibit more flakiness" means. Should we clarify? Is it error rate? Is it number of commits? And it may be better phrased like

> The table is limited to the 1000 flaky tests that exhibited the most flakiness